src/main/java/us/abstracta/wiresham/Flow.java

```diff
@@ -285,6 +296,14 @@ public static class FlowConstructor extends Constructor {
 
     @Override
     protected Object constructObject(Node node) {
+      if (node.getTag().getValue().equals("!parallel:")) {
```

Comment:
again collons, is this ok?

Reply:
I left the new design of the yaml in above comment. 
Regarding your second comment. No, it was not the cause of the issue. Since my first approach for the solution was using brackets as we did for `!server`, `!client` and `!include` and the issue persists. Custom constructor is needed due to the lack of parameters presented in the yaml.
The deserialization process require one parameter if the object to be constructed has a constructor with one parameter. The Yaml api requires the explicit parameter which could be something like `!parallel { parallelSteps: ...}` or even a list as we previously saw (`!parallel { - ...}`) The reason why `-` works to deserialize is based on the fact that the `-` is parsed as a SequenceNode which at the end does nothing else that provide the list of lists. It's an unnecessary node from my pov, but understandable from snake-yaml api side. 

For these scenarios snake-yaml could provide a simple api that won't require the specification of a parameter when dealing with a class that only has one field to deserialize.